app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,111 @@
+// Place all the styles related to the posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+#sign {
+  font-size: 30px;
+  position: absolute;
+  right: 20px;
+  margin-top: 44px;
+  color: #1D4B60;
+  text-decoration: none;
+  padding-right: 15px;
+  letter-spacing: 1px;
+ }
+
+div a {
+	text-align: center;
+	color: #1D4B60;
+	text-decoration: none;
+	padding: 0px 15px;
+	letter-spacing: 1px;
+}
+
+h3 a {
+	text-align: left;
+	color: #1D4B60;
+	text-decoration: none;
+	padding-left: 0px;
+	margin-bottom:2px;
+	letter-spacing: 1px;
+	font-size: 30px;
+}
+
+h3 {
+	text-align: left;
+	margin: 0px 0px;
+}
+
+img {
+	width: 600px;
+}
+
+#add {
+  width: 300px;
```

Comment:
Properties should be ordered border, border-radius, font-size, margin, margin-bottom, margin-top, text-align, width